src/main/java/first/controllers/StudentControllerInter.java

```diff
@@ -0,0 +1,16 @@
+package first.controllers;
+
+public interface StudentControllerInter {
```

Comment:
I am not sure about this name, it is describing nothing. Could you rename it to something else? For example BaseHtmlForm